langchain4j-qianfan/src/main/java/dev/langchain4j/model/qianfan/QianfanModelEnum.java

```diff
@@ -0,0 +1,60 @@
+package dev.langchain4j.model.qianfan;
+
+import lombok.Getter;
+
+/**
+ * Qianfan model, the most perfect model in China at present
+ */
+@Getter
+public enum QianfanModelEnum {
```

Comment:
Could you please split this into 3 enums?
```
QianfanChatModelName
QianfanLanguageModelName
QianfanEmbeddingModelName
```
So that it is clearer to users which one to use with which model class